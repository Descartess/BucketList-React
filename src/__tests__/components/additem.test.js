import React from 'react';
import renderer from 'react-test-renderer';
import AddBucketListItemForm from '../../components/additem';

describe('add item component', () => {
  it('renders correctly', () => {
    const rendered = renderer.create(
      <AddBucketListItemForm />,
    );
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
