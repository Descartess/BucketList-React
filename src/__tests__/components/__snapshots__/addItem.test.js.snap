// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add bucketlist item snapshot  Add Bucketlist Item form renders correctly 1`] = `null`;
